packages/react-vapor/src/components/input/LabeledInput.tsx

```diff
@@ -34,11 +35,11 @@ export const LabeledInput: React.FunctionComponent<ILabeledInputProps> = ({
         ) : null;
 
     return (
-        <>
+        <Section level={3}>
```

Comment:
UX previously recommended against this change since it could be wrapper on level 3 or 4, are inputs always on level 3 now...?

Reply:
since @Charles-ErickBG want almost everything to fall in the right place from the get go, it seemed an easy way.
Do you think you should do it otherwise ?

Reply:
I mean... my initial design suggested three levels:

`Form > FormGroup > Input`

Then it was changed to `Section` to allow having `Section (1) > Section (2) > Section (3) > Input` as well as `Section (1) > Section (2) > Input` as well as `Section (1) > Input`.

Having a level 3 hardcoded into the `LabeledInput` kind of bring it back to the original design and enforces 3 levels.

I mean.. I'm fine with it, but I would have kept the `Form > FormGroup > Input` structure if that's the use case now 😄

Reply:
When you put it this way, I think that you designed it like this for more flexibility. We should respect that, I totally agree. It is important that we all learn the pattern correctly to use the components in the good way so it will be easier for everyone. and maybe this is what I missed with the labeled Input.